js/app/application.js

```diff
@@ -43,6 +43,19 @@ const KnowledgeSearchIface = '\
   </interface> \
 </node>';
 
+const GrandCentralContentIface = '\
+<node> \
+  <interface name="com.endlessm.GrandCentralContent"> \
+    <method name="ContentQuery"> \
+      <arg type="s" name="Query" direction="in" /> \
+      <arg type="s" name="Result" direction="out" /> \
+    </method> \
+    <method name="QueryNotifications"> \
+      <arg name="notifications" direction="out" type="aa{ss}" /> \
+    </method> \
```

Comment:
Ah yes, I had this in there in the past, since I wanted to query the shell itself for historical notifications instead of handling new ones. Also, the code used to handle all the interactions for notification is ... complicated and it seemed easier to just re-use them. But nonetheless, notifications are not in scope right now, so I'll get rid of this.